bigquery/README.md

```diff
@@ -111,10 +111,15 @@ deactivate
 When indexing a large table on a Mac, Elasticsearch may crash with no error
 message in the logs. Try increasing Docker's memory, for example from 2G to 3G.
 
-### Tips for indexing large tables locally
+### Tips for working locally with large tables
 
-A 2G table can take 4 hours to index. You don't want to
-accidentally wipe your Elasticsearch index and have to reindex.
+A 2G table can take 4 hours to index. Here are tips so you don't have to wait
```

Comment:
Throughout this doc, can we use GB instead of G? (assuming you're referring to gigabytes)